README.md

```diff
@@ -50,6 +50,6 @@ $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/maste
 - `$ pipenv lock` (Lock the environment to ensure the same dependency for production)
 
 # How to run service
-- On the command, type `$ pipenv install --ignore-pipfile`. This ignores the Pipfile for installation and use what's in the Pipfile.lock
-- Type `$ pipenv run python nameofthefile.py`. Example: `$ pipenv run python main.py`
+- On the command, type `$ pipenv install`. This ignores the Pipfile for installation and use what's in the Pipfile.lock
+- Type `$ pipenv run python3 main.py`. 
```

Comment:
- [ ] The service under the **Purpose** section is wrong
- [ ]  `wget` installation needs to be documented
- [ ] `generate_dep.sh` needs to be documented